index.js

```diff
@@ -21,11 +21,11 @@ const parseValue = (value, href) => {
 
 // Get `<a>` element as string
 function linkify(href, options = {}) {
-	// The regex URL mistakenly includes a dot at the end of the URL
-	let trailingDot = '';
-	if (href.endsWith('.')) {
+	// The regex URL mistakenly includes punctuation (a period or question mark) at the end of the URL
+	let punctuation = '';
```

Comment:
Let's just use a regex to extract it here, like:

``` JavaScript 
const punctuation = trailingPunctuationRegex.exec(href) ?? '';
if (punctuation) {
  href = href.slice(0, -1);
}

Reply:
Done in 669b814789c43d341e7687a23f64d5034973f630